File: AnuitexTraining.BusinessLogicLayer/Models/Orders/OrderItemModel.cs
using AnuitexTraining.BusinessLogicLayer.Models.Base;
using AnuitexTraining.BusinessLogicLayer.Models.PrintingEditions;
using static AnuitexTraining.Shared.Enums.Enums;

namespace AnuitexTraining.BusinessLogicLayer.Models.Orders
{
    public class OrderItemModel : BaseModel
    {
        public double Amount { get; set; }
        public CurrencyType Currency { get; set; }
        public long PrintingEditionId { get; set; }
        public PrintingEditionModel PrintingEdition { get; set; }
        public long OrderId { get; set; }
        public int Count { get; set; }
    }
}

File: AnuitexTraining/ClientApp/src/app/modules/cart/models/order-item.model.ts
import {CurrencyTypeEnum} from "../../shared/enums/currency-type.enum";

export interface OrderItemModel {
  id: number
  amount: number,
  currency: CurrencyTypeEnum,
  printingEditionId: number,
  orderId: number,
  count: number,
  title: string
}
